person.rb

```diff
@@ -0,0 +1,85 @@
+require_relative 'rental'
+
+class Nameable
+  attr_accessor :name
```

Comment:
[OPTIONAL] I highly recommend that you move the `Nameable`, `BaseDecorator`, `CapitalizeDecorator`, and `TrimmerDecorator` classes to their own separate files. It is best to have one class in a file to avoid having clustered code in one file.

Reply:
😄 Done